doc/en/user/source/security/layer.rst

```diff
@@ -14,16 +14,19 @@ Providing access to layers is linked to :ref:`roles <security_rolesystem_roles>`
 Rules
 -----
 
-The syntax for a layer security rule is as follows (``[]`` denotes optional
-parameters)::
+The syntax for a layer security rule can follow three different patterns (``[]`` denotes optional parameters)::
 
+  globalLayerGroup.permission=role[,role2,...]
   workspace.layer.permission=role[,role2,...]
+  workspace.workspaceLayerGroup.permission=role[,role2,...]
 
```

Comment:
May be worth make it clear that a globalLayerGroup is just a a layerGroup without a workspace.